chain/client/src/client_actor.rs

```diff
@@ -1683,6 +1687,18 @@ impl Handler<StateSplitResponse> for ClientActor {
     }
 }
 
+/// Returns random seed sampled from the current thread
+pub fn random_seed_from_thread() -> RngSeed {
+    let mut rng_seed: RngSeed = [0; 32];
+    for j in 0..3 {
```

Comment:
can you explain why do you want to create a seed from concatenating multiple values? 

RngSeed doesn't offer any other constructor?